vtex/faststore
# SEO | Breadcrumb structured data

## What's the purpose of this pull request?
Adds breadcrumb to search pages

## How it works? 
Also, on all static generated pages this adds a breadcrumb RichResult so google can index and interpret the data correctly

## How to test it?
[btglobal](https://github.com/vtex-sites/btglobal.store/pull/10)
[storecomponents](https://github.com/vtex-sites/storecomponents.store/pull/447)
[marinbrasil](https://github.com/vtex-sites/marinbrasil.store/pull/316)

Open [Google Rich Results](https://search.google.com/test/rich-results) tool and paste a category from any of the deploy previews below. This check should pass for category pages and fail for fulltext or pages with filters

## References
<em>Spread the knowledge: is there any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: adding references to related issues or mentioning people important to this PR may be good for the documentation and reviewing process</em>
